solutions/kubernetes/io-sanity.sh

```diff
@@ -132,7 +132,20 @@ function run_io_sanity() {
    add_primary_separator "\tSuccessfully Passed IO Sanity Testing"
 }
 
+function run_data_io_sanity() {
+   kubectl exec -it $(kubectl get pods | awk '/cortx-client/{print $1; exit}') -c cortx-hax -- bash -c "pip3 install pandas xlrd \
```

Comment:
I feel Motr team should add a script for this and we can just call it. So that they can control parameters.

Reply:
Yeah, can ask them for script